README.md

```diff
@@ -373,18 +356,36 @@ All images are pushed to [Docker Hub](docker_hub_tags).
 
 Fixed version tags (`x.y.z`) are frozen: we never update them.
 
+Example of minor versions timeline:
+
+- `x.y.0` - Alpha
+-   | -> `x.y.1` - Beta
+-     | -> `x.y.2` - Stable
+-       | -> `x.y.3` - Update
+
 Rolling versions are updated to the last fixed version tags:
 
 - `x.y` == `x.y.<last-z>` (`==` means 'points to the same image')
 - `1` == `1.<last-y>.<last-z>`
 - `2` == `2.<last-y>.<last-z>`
 - `latest` == `2`
 
+Examples of stable rolling versions:
+
+- `1` -> `1.<last-y>.2`
+- `2` -> `2.<last-y>.2`
+- `latest` -> `2.<last-y>.2`
```

Comment:
It seems it feeds the save information as the paragraps above.

Reply:
My bad. After discussion with Alexander I see the difference. Here we have explicit `.2` at the end.

I vote to just replace the `.<last-z>` with `.2` sbove.